kosmos/docs/v0.2.0/tutorials/design-of-exec-and-log.md

```diff
@@ -238,5 +238,5 @@ By default, InternalDNS has the highest priority.
 ![EXE_Log_3.png](img/EXE_Log_3.png)

```

Comment:
Wouldn't it be impressive if we used console output here?

Reply:
[root@kosmos-control-cluster ~]# kubectl get nodes -owide
NAME               STATUS   ROLES                         AGE   VERSION   INTERNAL-IP    EXTERNAL-IP   OS-IMAGE                KERNEL-VERSION               CONTAINER-RUNTIME
kosmos-control-1   Ready    worker                        65d   v1.21.5   192.168.0.1    <none>        CentOS Linux 7 (Core)   3.10.0-957.27.2.el7.x86_64   docker://20.10.8
kosmos-control-2   Ready    control-plane,master,worker   65d   v1.21.5   192.168.0.2    <none>        CentOS Linux 7 (Core)   3.10.0-957.27.2.el7.x86_64   docker://20.10.8
kosmos-cluster7    Ready    agent                         20d   v1.21.5   192.168.0.3    <none>        <unknown>               <unknown>                    <unknown>